ui/src/main/java/com.github.raulccabreu.redukt.ui/StateListenerLayout.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.raulccabreu.redukt.ui
+
+import com.github.raulccabreu.redukt.Redukt
+import com.github.raulccabreu.redukt.states.StateListener
+
+interface StateListenerLayout<T> : StateListener<T> {
+
+    fun getRedukt(): Redukt<T>
+
+    fun start() {
```

Comment:
the names start and stop looks too generic also for me. I guess that we should name them more appropriated. Maybe the old registerState and unregisterState is more meaningful.

Reply:
registerState or registerListener ?

Reply:
what about registerStateListener ?